commando/commando.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+"""Commando is a plugin to allow one node to control another.  You use
+"commando" to send commands, and the 'commando-writer' and
+'commando-reader' to allow nodes to send you commands.
+
+The formats are:
+
+type:4C4F - execute this command
+type:594B - reply (with more coming)
+type:594D - last reply
+
+Each one is an 8 byte id (to link replies to command), followed by JSON.
+"""
+from pyln.client import Plugin, RpcError
+import json
+import textwrap
+import random
+
+plugin = Plugin()
+
+# "YOLO"!
+COMMANDO_CMD = 0x4c4f
+
+# Replies are split across multiple CONTINUES, then TERM.
+COMMANDO_REPLY_CONTINUES = 0x594b
+COMMANDO_REPLY_TERM = 0x594d
+
+
+class CommandResponse:
+    def __init__(self, req):
+        self.buf = bytes()
+        self.req = req
+
+
+def split_cmd(cmdstr):
+    """Interprets JSON and method and params"""
+    cmd = json.loads(cmdstr)
+
+    return cmd['method'], cmd.get('params')
+
+
+def send_msg(plugin, peer_id, msgtype, idnum, contents):
+    """Messages are form [8-byte-id][data]"""
+    msg = (msgtype.to_bytes(2, 'big')
+           + idnum.to_bytes(8, 'big')
+           + bytes(contents, encoding='utf8'))
+    plugin.rpc.call(plugin.msgcmd, {'node_id': peer_id, 'msg': msg.hex()})
+
+
+def send_result(plugin, peer_id, idnum, res):
+    # We can only send 64k in a message.
+    parts = textwrap.wrap(json.dumps(res), 65000)
```

Comment:
65000 seems like a *magic number* and in the test is another *magic number*, but then it's 65535.
The latter is close to `64*1024=65536`, but not the same.
Maybe use a constant and do `CONSTANT -1` where appropriate (with comment)?

Reply:
I will define it.  The entire packet can be 65535, but there's overhead in the message itself.

Thanks!